io/OGRWriter.cpp

```diff
@@ -60,7 +60,7 @@ static StaticPluginInfo const s_info
 CREATE_STATIC_STAGE(OGRWriter, s_info)
 
 OGRWriter::OGRWriter() : m_driver(nullptr), m_ds(nullptr), m_layer(nullptr),
-    m_feature(nullptr)
+    m_feature(nullptr), m_curCount(0), m_measureDim(Dimension::Id::Unknown)
```

Comment:
`m_curCount` should be initialized in `readyFile()`, not in the ctor. This makes sure that it's properly set if the writer is used more than once.